<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewGameDialog</class>
 <widget class="QDialog" name="NewGameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>358</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Create Game</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Images.qrc">
    <normaloff>:/images/gomoku_icon.ico</normaloff>:/images/gomoku_icon.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxMode">
     <property name="title">
      <string>Mode</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QComboBox" name="comboGameType">
        <item>
         <property name="text">
          <string>Player vs Player</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Player vs A.I.</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>A.I. vs A.I.</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboDificulty">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Easy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Medium</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hard</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkFirstPlayer">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>A.I. play first</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="minimaxDebug">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>MiniMax debug</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxRules">
     <property name="title">
      <string>Rules</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="checkDoubleThree">
        <property name="text">
         <string>Double three enabled</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkEndGameTaking">
        <property name="text">
         <string>End game taking enabled</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
